src/directory-sync/interfaces/list-directories-options.interface.ts

```diff
@@ -5,3 +5,9 @@ export interface ListDirectoriesOptions extends PaginationOptions {
   organizationId?: string;
   search?: string;
 }
+
+export interface SerializedListDirectoriesOptions extends PaginationOptions {
```

Comment:
Should we have a separate `SerializedPaginationOptions`?

Reply:
It's not really serializing them since there's no diff. I found that to be pretty weird?

Reply:
What if we were to add a new field to `PaginationOptions` in the future that did need to be serialized?

Reply:
So your take is that we should have a no-op serialization for all pagination endpoints in case we add this in the future? I can see this as a safeguard though I'm not extremely driven to add it personally. If you'd feel safer with it though happy to add.